traffic_portal/app/src/index.html

```diff
@@ -53,6 +53,8 @@
         <script src="resources/assets/js/moment-min_2.22.1.js"></script>
         <script src="resources/assets/js/underscore-min_1.8.3.js"></script>
         <script src="resources/assets/js/jquery.dataTables.min_1.10.16.js"></script>
+        <script src="resources/assets/js/chartjs/Chart.min.js"></script>
```

Comment:
can u add the versions to the file names? see how the others have version numbers?